accepted/0000-support-script-files-in-npm-run.md

```diff
@@ -0,0 +1,53 @@
+# Script file fallback in `npm run`
+
+## Summary
+
+`package.json` scripts are very handy. We can rely them to create industry standards like `npm run build` and `npm run test` to build and test files respectively.
+This RFC suggests a fallback for script files, instead of explicitly listing them in `scripts`.
+
+## Motivation
+
+In big Node.js repos, the `package.json#scripts` JSON key might get very crowded. It's widely common to see `test:e2e`, `test:unit`, `test:integration` and many more things.
+The problem is that declaring scripts in JSON is hard to maintain and to reason about:
+
+- Any logic you put into the scripts is hard to manage, because they're practically one-liners
+- You can't use prettier/shellcheck/TypeScript or any other tool to check your syntax and enforce standards
+- If you do want to extract your script into a standalone script file, you will still need to have an explicit reference to the file from `package.json#scripts`
+
+npm could suggest a standard location to write script files so they will be picked up automatically instead of explicitly referencing them in `package.json#scripts`.
```

Comment:
The problems you list seem to be solved by an explicit call out to a separate script file - which means this RFC is only solving the problem of "explicitly adding a `scripts` entry"?